ci-operator/config/ViaQ/logging-kibana/ViaQ-logging-kibana-openshift-logging.yaml

```diff
@@ -0,0 +1,255 @@
+base_images:
+  cluster-logging-operator-5-5:
+    name: "5.5"
+    namespace: logging
+    tag: cluster-logging-operator
+  cluster-logging-operator-5-6:
+    name: "5.6"
+    namespace: logging
+    tag: cluster-logging-operator
+  cluster-logging-operator-5-7:
+    name: "5.7"
+    namespace: logging
+    tag: cluster-logging-operator
+  cluster-logging-operator-e2e-5-5:
+    name: "5.5"
+    namespace: logging
+    tag: cluster-logging-operator-e2e
+  cluster-logging-operator-e2e-5-6:
+    name: "5.6"
+    namespace: logging
+    tag: cluster-logging-operator-e2e
+  cluster-logging-operator-e2e-5-7:
+    name: "5.7"
+    namespace: logging
+    tag: cluster-logging-operator-e2e
+  cluster-logging-operator-registry-5-5:
+    name: "5.5"
+    namespace: logging
+    tag: cluster-logging-operator-registry
+  cluster-logging-operator-registry-5-6:
+    name: "5.6"
+    namespace: logging
+    tag: cluster-logging-operator-registry
+  cluster-logging-operator-registry-5-7:
+    name: "5.7"
+    namespace: logging
+    tag: cluster-logging-operator-registry
+  elasticsearch-operator-5-5:
+    name: "5.5"
+    namespace: logging
+    tag: elasticsearch-operator
+  elasticsearch-operator-5-6:
+    name: "5.6"
+    namespace: logging
+    tag: elasticsearch-operator
+  elasticsearch-operator-5-7:
+    name: "5.7"
+    namespace: logging
+    tag: elasticsearch-operator
+  elasticsearch-operator-registry-5-5:
+    name: "5.5"
+    namespace: logging
+    tag: elasticsearch-operator-registry
+  elasticsearch-operator-registry-5-6:
+    name: "5.6"
+    namespace: logging
+    tag: elasticsearch-operator-registry
+  elasticsearch-operator-registry-5-7:
+    name: "5.7"
+    namespace: logging
+    tag: elasticsearch-operator-registry
+  elasticsearch-proxy:
+    name: 5.y
+    namespace: logging
+    tag: elasticsearch-proxy
+  log-file-metric-exporter:
+    name: 5.y
+    namespace: logging
+    tag: log-file-metric-exporter
+  logging-elasticsearch6:
+    name: 5.y
+    namespace: logging
+    tag: logging-elasticsearch6
+  logging-eventrouter:
+    name: 5.y
+    namespace: logging
+    tag: logging-eventrouter
+  logging-fluentd:
+    name: v1.14.x
+    namespace: logging
+    tag: logging-fluentd
+  oauth-proxy:
+    name: "4.9"
+    namespace: ocp
+    tag: oauth-proxy
+  ubi8:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-base-openshift-4.12
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-golang-1.19-openshift-4.12
+images:
+- dockerfile_path: Dockerfile
+  from: ubi8
+  to: logging-kibana6
+promotion:
+  name: 6.8.1
```

Comment:
Does it make sense to continue to promote this to `logging/5.y:logging-kibana6`, so that it just replaces the existing kibana image?

Related to this: Do we still want to build kibana from origin-aggregated-logging or will this version be "the only one available in CI"?